src/Gui/MainWindow.cpp

```diff
@@ -1917,13 +1917,15 @@ void MainWindow::startSplasher()
             GetGroup("BaseApp")->GetGroup("Preferences")->GetGroup("General");
         // first search for an external image file
         if (hGrp->GetBool("ShowSplasher", true)) {
-            d->splashscreen = new SplashScreen(this->splashImage());
+            d->splashscreen = new SplashScreen(this->splashImage(), 
+                Qt::WindowStaysOnTopHint | Qt::SplashScreen | Qt::X11BypassWindowManagerHint);
```

Comment:
`Qt::SplashScreen` is already set in [QSplashScreen](https://codebrowser.dev/qt5/qtbase/src/widgets/widgets/qsplashscreen.cpp.html#_ZN13QSplashScreenC1ERK7QPixmap6QFlagsIN2Qt10WindowTypeEE) constructor. Also there is a trailing whitespace added.

Reply:
Ok, fixed